src/ui/link/index.tsx

```diff
@@ -83,13 +87,24 @@ const Link = forwardRef<HTMLButtonElement | HTMLAnchorElement, LinkProps>(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [route, linkDestination, routerActions]);
 
+    const schedulePrefetch = useCallback(() => {
+      prefetchRef.current = setTimeout(triggerPrefetch, PREFETCH_DELAY);
```

Comment:
Should we check that `current` is not set yet? I wonder if there is a risk that `schedulePrefetch` is called twice and so we end up with 2 timers being set, but only one getting cancelled 🤔